.buildkite/JuliaProject.toml

```diff
@@ -1,8 +1,12 @@
 [extras]
+CUDA_Driver_jll = "4ee394cb-3365-5eb0-8335-949819d2adfc"
 CUDA_Runtime_jll = "76a88914-d11a-5bdc-97e0-2f5a05c973a2"
 HDF5_jll = "0234f1f7-429e-5d53-9886-15a909be8d59"
 MPIPreferences = "3da0fdf6-3ccc-4f1b-acd9-58baa6c99267"
 
+[preferences.CUDA_Driver_jll]
+ compat = false
```

Comment:
Can you please remove the whitespace before `compat`?

Reply:
yes, good catch